lib/puppet/network/formats.rb

```diff
@@ -151,7 +151,7 @@ def render(datum)
     end
 
     # ...or pretty-print the inspect outcome.
-    return json.render(datum)
+    return JSON.pretty_generate(datum)
```

Comment:
Puppet's `json.render` method uses PSON which monkey patches `to_pson` on a number of classes: `Object`, `Hash`, `Array`, `Integer`, `Float`, `String`, `TrueClass`, `FalseClass`, `NilClass`. The `:console` renderer short-circuits evaluation of [`String` and `Numeric`](https://github.com/puppetlabs/puppet/blob/master/lib/puppet/network/formats.rb#L127-L128), but not `Object`, `TrueClass`, `FalseClass`, `NilClass`. Also when converting Hash, the renderer is still calling `json.render` for its [nested values](https://github.com/puppetlabs/puppet/blob/master/lib/puppet/network/formats.rb#L136).

I'm thinking we either need to switch everything over to using `JSON.pretty_generate` and ensure the console renderer short-circuits the other classes that PSON monkey patches, e.g. `TrueClass`.

Or we go back to `json.render(datum)` and deal with eliminating PSON another day.

As much as I don't like PSON, I think it would be cleaner and less risky to do the latter, and then remove all of PSON in one PR at a future date.